docs/doc/11-integrations/10-data-tool/00-vector.md

```diff
@@ -1,73 +1,61 @@
 
-Databend supports ClickHouse REST API, so it's easy to integration with Vector to stream, aggregate, and gain insights.
+Vector natively supports delivering data to [Databend as a sink](https://vector.dev/docs/reference/configuration/sinks/databend/), this means that Vector can send data to Databend for storage or further processing. Databend acts as the destination for the data collected and processed by Vector. By configuring Vector to use Databend as a sink, you can seamlessly transfer data from Vector to Databend, enabling efficient data analysis, storage, and retrieval. 
 
-## Create a Databend User
+## Integrating with Vector
 
-Connect to Databend server with MySQL client:
-```shell
-mysql -h127.0.0.1 -uroot -P3307 
-```
+To integrate Databend with Vector, start by creating an SQL account in Databend and assigning appropriate permissions. This account will be used for communication and data transfer between Vector and Databend. Then, in the Vector configuration, set up Databend as a Sink.
+
+### Step 1: Creating a SQL User in Databend
+
+For instructions on how to create a SQL user in Databend and grant appropriate privileges, see [Create User](../../14-sql-commands/00-ddl/30-user/01-user-create-user.md). Here's an example of creating a user named *user1* with the password *abc123*:
 
-Create a user:
 ```sql
 CREATE USER user1 IDENTIFIED BY 'abc123';
-```
 
-Create a Database:
-```sql
 CREATE DATABASE nginx;
-```
 
-Grant insert privileges for the user:
-```sql
 GRANT INSERT ON nginx.* TO user1;
 ```
 
-See also [How To Create User](../../14-sql-commands/00-ddl/30-user/01-user-create-user.md).
+### Step 2. Configure Databend as a Sink in Vector
 
-## Configure Vector
+In this step, configure Databend as a sink in Vector by specifying the necessary settings such as the input source, compression, database, endpoint, table, and authentication credentials (username and password) for Databend integration. The following is a simple example of configuring Databend as a sink. For a comprehensive list of configuration parameters, refer to the Vector documentation at https://vector.dev/docs/reference/configuration/sinks/databend/
 
-To use Databend with Vector you will need to configure for [Clickhouse Sink](https://vector.dev/docs/reference/configuration/sinks/clickhouse/#example-configurations):
+```toml title='vector.toml'
+...
 
-```shell
 [sinks.databend_sink]
-type = "clickhouse"
+type = "databend"
 inputs = [ "my-source-or-transform-id" ] # input source
-// highlight-next-line
-database = "mydatabase" #Your database
-// highlight-next-line
-table = "mytable" #Your table.
-// highlight-next-line
-endpoint = "http://localhost:8124/" #Databend ClickHouse REST API: http://{clickhouse_http_handler_host}:{clickhouse_http_handler_port}/
-compression = "gzip"
-```
+compression = "none"
+database = "nginx" #Your database
+endpoint = "http://localhost:8000"
+table = "mytable" #Your table
+
+...
 
-```shell
 [sinks.databend_sink.auth]
```

Comment:
I think this does not work, we need a dsn here? cc @everpcpc 
We can try to deploy a nginx and test it follow the <Analyzing Nginx Access Logs with Databend> guide line :)

Reply:
Vector sink currently use these settings, DSN could be supported later.